.storybook/config.js

```diff
@@ -62,6 +64,16 @@ addParameters({
       iframes: true,
     },
   },
+  docs: {
```

Comment:
Part of the fix for

> 1. Components aren't re-rendering when navigating around Storybook, or modifying knobs.

Which is all just removing `addon-notes` and using `addon-docs` instead